js/ad_manager_pulse.js

```diff
@@ -0,0 +1,1110 @@
+            };
+
+            var _onAdSkipped = function(){
+                amc.notifyLinearAdEnded(1);
+            };
+
+            var _onAdBreakFinished = function(){
+                this._currentAdBreak = null;
+                this.notifyAdPodEnded();
+            };
+
+            var _onAdBreakStarted = function(event,eventData) {
+                this._adPlayer.resize(-1,
+                    -1, this._isFullscreen);
+                this._currentAdBreak = eventData.adBreak;
+                this.notifyAdPodStarted(this._adBreakId,this._currentAdBreak.getPlayableAdsTotal());
+
+            };
+
+            var _onAdClicked = function (event,eventData) {
+                this.videoControllerWrapper.togglePlayPause();
+            };
+            var _onAdPaused = function(event,metadata){
+                this.videoControllerWrapper.raisePauseEvent();
+            }
+
+            var _onAdPlaying = function(event,metadata){
+                this.videoControllerWrapper.raisePlayingEvent();
+            }
+            var _onAdTimeUpdate = function(event, eventData) {
+
+                var duration = eventData.duration ? eventData.duration :this.currentAd.getCoreAd().creatives[0].duration;
+                this.videoControllerWrapper.raiseTimeUpdate(eventData.position, duration);
+            };
+
+            var _onAdStarted = function(event, eventData) {
+                this._currentAd = eventData.ad;
+
+                var clickThroughURL =  this._currentAd.getClickthroughURL();
+                var skipOffset = this._currentAd.getSkipOffset();
+                var name = null;
+
+                if(this._showAdTitle){
+                    name = this._currentAd.getCoreAd().title;
+                }
+
+                amc.notifyLinearAdStarted(1,
+                    {duration: this._currentAd.getCoreAd().creatives[0].duration,
+                        name: name,
+                        indexInPod: eventData.adPosition,
+                        skippable:this._currentAd.isSkippable(),
+                        hasClickUrl: clickThroughURL ? true : false});
+
+                if(this._currentAd.isSkippable()) {
+                    amc.showSkipVideoAdButton(true, skipOffset.toString());
+                }
+                else {
+                    amc.showSkipVideoAdButton(false);
+                }
+                this._adPlayer.resize(-1,
+                    -1, this._isFullscreen);
+            };
+        }
+        return new PulseAdManager();
+    });
+
+    //Pulse Video plugin
+    var PulsePlayerFactory = function() {
+        this.adManager = {};
+        this.name = "PulseVideoTech";
+        this.encodings = [OO.VIDEO.ENCODING.IMA];
+        this.features = [OO.VIDEO.FEATURE.VIDEO_OBJECT_SHARING_TAKE];
+        this.technology = OO.VIDEO.TECHNOLOGY.HTML5 ;
+        // This module defaults to ready because no setup or external loading is required
+        this.ready = true;
```

Comment:
we later removed this property so you can remove it if you would like.  It wont' be used